static/tables/CoordinateOperationMethods.html

```diff
@@ -23,8 +23,8 @@ <h1>Apache SIS™ Coordinate Operation Methods</h1>
           The <code>semi-major</code> and <code>semi-minor</code> parameters are needed for all map projections,
           but usually do not need to be specified explicitly since they are inferred from the ellipsoid
           (unless <a href="http://sis.apache.org/apidocs/org/apache/sis/referencing/operation/transform/DefaultMathTransformFactory.html">creating parameterized transforms directly</a>).
-          For this reason, those parameters are usually not shown in <a href="https://epsg.org/">EPSG registry</a>
-          or <a href="https://docs.ogc.org/is/12-063r5/12-063r5.html">Well Known Text</a> (WKT) definitions.
+          For this reason, those parameters are usually not shown in <a href="http://www.epsg-registry.org">EPSG registry</a>
+          or <a href="http://docs.opengeospatial.org/is/12-063r5/12-063r5.html">Well Known Text</a> (WKT) definitions.
```

Comment:
More generally, maybe we should revert all changes in the `static/*.html` files… Since there is no reason for those files to change, all differences are probably edition added after the port to Hugo.